pkg/cluster/managed.go

```diff
@@ -76,14 +78,7 @@ func (c *Cluster) assignManagedDriver(ctx context.Context) error {
 		}
 	}
 
-	endpointType := strings.SplitN(c.config.Datastore.Endpoint, ":", 2)[0]
-	for _, driver := range managed.Registered() {
-		if endpointType == driver.EndpointName() {
-			c.managedDB = driver
-			return nil
-		}
-	}
```

Comment:
This looked like it was intended to work with the code above, but didn't seem to actually be used since we always either have an initialized datastore, or are initializing/joining an existing cluster using the default driver.

Reply:
I think that code was brought over from kine where we use http:// and https:// endpoints for etcd, but you are correct i dont think its needed here

Reply:
yeah, for managed etcd EndpointName() returns "etcd" so this would have looked for an etcd:// string which I have never seen used.